src/Cosserat/mapping/DiscreteCosseratMapping.inl

```diff
@@ -72,6 +72,8 @@ DiscreteCosseratMapping<TIn1, TIn2, TOut>::DiscreteCosseratMapping()
 template <class TIn1, class TIn2, class TOut>
 void DiscreteCosseratMapping<TIn1, TIn2, TOut>::init()
 {
+    Inherit1::init();
```

Comment:
@adagolodjo @alxbilger This change makes the scenes crash.

```console
########## SIG 11 - SIGSEGV: segfault ##########
  sofa::helper::BackTrace::sig(int)
  sofa::component::mapping::BaseCosserat<sofa::defaulttype::StdVectorTypes<sofa::type::Vec<3u, double>, sofa::type::Vec<3u, double>, double>, sofa::defaulttype::StdRigidTypes<3u, double>, sofa::defaulttype::StdRigidTypes<3u, double> >::update_ExponentialSE3(sofa::type::vector<sofa::type::Vec<3u, double>, sofa::type::CPUMemoryManager<sofa::type::Vec<3u, double> > > const&)
``` 

It seems that the vectors `m_indicesVectors`, `m_framesLengthVectors`, and `m_BeamLengthVectors` are not correctly initialized before the call to `update_ExponentialSE3()` .